lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -3361,20 +3360,22 @@ def test_permuations(self):
         """Test every permutation"""
         old_course_key = CourseKey.from_string("OpenEdX/Old/2020")
         new_course_key = CourseKey.from_string("course-v1:OpenEdX+New+2020")
-        global_staff_user = UserFactory(username="global_staff", is_staff=True)
-        regular_user = UserFactory(username="normal", is_staff=False)
```

Comment:
Except where noted, the same set of changes is being made for all the assertions in this test function:
* just pass True or False in for is_global_staff, instead of a user whose is or isn't global staff.
* function was renamed from `show_courseware_mfe_link` to `courseware_mfe_is_advertised`.
* the course key is now the first argument instead of the third.

Reply:
All improvements. 👍🏼